api-schema/src/main/java/org/stellar/anchor/api/custody/fireblocks/TransactionStatus.java

```diff
@@ -0,0 +1,35 @@
+package org.stellar.anchor.api.custody.fireblocks;
+
+import java.util.Set;
+
+public enum TransactionStatus {
```

Comment:
❓ Are these `fireblocks` specific statuses?

Reply:
Yes. This enum contains all possible statuses of `fireblocks` transaction according to https://docs.fireblocks.com/api/?python#transactionstatus